[2025-08-25T14:47:42.485+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mamda_monthly_etl_pipeline.check_status_files manual__2025-08-25T14:47:39.431016+00:00 [queued]>
[2025-08-25T14:47:42.495+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mamda_monthly_etl_pipeline.check_status_files manual__2025-08-25T14:47:39.431016+00:00 [queued]>
[2025-08-25T14:47:42.496+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-08-25T14:47:42.510+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): check_status_files> on 2025-08-25 14:47:39.431016+00:00
[2025-08-25T14:47:42.516+0000] {standard_task_runner.py:60} INFO - Started process 1500 to run task
[2025-08-25T14:47:42.519+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mamda_monthly_etl_pipeline', 'check_status_files', 'manual__2025-08-25T14:47:39.431016+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/mamda_monthly_etl.py', '--cfg-path', '/tmp/tmpdcjtsgie']
[2025-08-25T14:47:42.521+0000] {standard_task_runner.py:88} INFO - Job 95: Subtask check_status_files
[2025-08-25T14:47:42.576+0000] {task_command.py:423} INFO - Running <TaskInstance: mamda_monthly_etl_pipeline.check_status_files manual__2025-08-25T14:47:39.431016+00:00 [running]> on host 9f5b6f3413d6
[2025-08-25T14:47:42.667+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mamda_etl' AIRFLOW_CTX_DAG_ID='mamda_monthly_etl_pipeline' AIRFLOW_CTX_TASK_ID='check_status_files' AIRFLOW_CTX_EXECUTION_DATE='2025-08-25T14:47:39.431016+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-25T14:47:39.431016+00:00'
[2025-08-25T14:47:42.668+0000] {mamda_monthly_etl.py:103} INFO - === CHECK: STATUS FILES ===
[2025-08-25T14:47:42.669+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.11/site-packages/***/utils/context.py:315: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2025-08-25T14:47:42.671+0000] {mamda_monthly_etl.py:118} WARNING - Status directory not found: /opt/***/data/fee_payouts_status
[2025-08-25T14:47:42.683+0000] {python.py:202} INFO - Done. Returned value was: False
[2025-08-25T14:47:42.704+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=mamda_monthly_etl_pipeline, task_id=check_status_files, execution_date=20250825T144739, start_date=20250825T144742, end_date=20250825T144742
[2025-08-25T14:47:42.732+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-25T14:47:42.757+0000] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
